packages/redux/src/products/types/actions/productOutfits.types.ts

```diff
@@ -0,0 +1,35 @@
+import type * as actionTypes from '../../actionTypes';
+import type { Action } from 'redux';
+import type { BlackoutError } from '@farfetch/blackout-client';
+import type { NormalizedSchema } from 'normalizr';
+import type { ProductEntity } from '../../../entities/types';
+
+export interface FetchProductOutfitsRequestAction extends Action {
+  meta: { productId: ProductEntity['id'] };
+  type: typeof actionTypes.FETCH_PRODUCT_OUTFITS_REQUEST;
+}
+
+export interface FetchProductOutfitsSuccessAction extends Action {
+  meta: { productId: ProductEntity['id'] };
+  payload: NormalizedSchema<
+    {
+      products: Record<ProductEntity['id'], ProductEntity>;
```

Comment:
The type of the value is not correct I believe. Can you check?

Reply:
I think this is right because the normalization is done inside the product entity.
We do the same on product sizes, product fittings, product details, etc..

Reply:
You are right, my mistake. Thanks!